packages/ilios-common/addon/components/course/objective-list-item.hbs

```diff
@@ -19,7 +19,7 @@
       </EditableField>
     {{else}}
       {{! template-lint-disable no-triple-curlies }}
-      {{{@courseObjective.title}}}
+      <FadeText @text={{{@courseObjective.title}}} />
```

Comment:
Since `<FadeText>` marks things as `htmlSafe` you can drop the triple currlies here and remove the lint disable line above.